src/index.ts

```diff
@@ -309,8 +306,11 @@ verifyButton.onclick = verifyCurrentCombination;
 
 restartButton.onclick = startNewGame;
 
-// add apply parameters event
-applyParametersButton.onclick = applyParameters;
+// add apply parameters event, usable with enter key to submit the form
+parametersPopup.addEventListener('submit', (e) => {
+  e.preventDefault();
+  applyParameters();
+});
```

Comment:
Can you move this listeners in the `listeners.ts `file ? Could be great to test it when we implement test ;) mb smthing like this : 

```ts
// listeners.ts

export addFormEvent(form:HTMLFormElement, action : ()=>void) { 
// implement code here
}

```

```ts
// index.ts
addFormEvent(parametersPopup,applyParameters)

Reply:
done

Reply:
Dont see the change ...